src/Integration/LocalServices/SolutionBindingInformationProvider.cs

```diff
@@ -31,13 +31,46 @@ public SolutionBindingInformationProvider(IServiceProvider serviceProvider)
         }
 
         #region ISolutionBindingInformationProvider
```

Comment:
Bug fixed: Previously would return that all the projects are unbound in an bound solution (test added)